jscomp/core/js_exp_make.ml

```diff
@@ -54,34 +54,41 @@ and is_pure_sub_exp (x : t) = remove_pure_sub_exp x = None
 (* let mk ?comment exp : t =
    {expression_desc = exp ; comment  } *)
 
-let var ?comment id : t = { expression_desc = Var (Id id); comment }
+let var ?comment id : t = { expression_desc = Var (Id id); comment; annotate_pure = false }
```

Comment:
I found the number of changes needed in this file surprising. Does it make sense to have a constructor?

```ocaml
let expression ?comment ?annotate_pure expression_desc : expression = ...
```